<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetadataWidgetSetting</class>
 <widget class="QWidget" name="MetadataWidgetSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>204</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>元数据来源：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="api_comboBox">
       <item>
        <property name="text">
         <string>网易云api</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>酷狗api</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>混合api</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="is_download_lrc_checkBox">
     <property name="text">
      <string>补全同时下载歌词</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="text">
      <string>》》》》》》</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="auto_button">
       <property name="text">
        <string>自动补全元数据</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>将会对载入的还未进行补全的文件
进行补全</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>379</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="sure_button">
       <property name="text">
        <string>确定设置</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
